client/components/Editor/plugins/table.ts

```diff
@@ -1,5 +1,22 @@
 import { keymap } from 'prosemirror-keymap';
-import { columnResizing, tableEditing, goToNextCell } from 'prosemirror-tables';
+// @ts-ignore
+import { columnResizing, tableEditing, goToNextCell, TableView } from 'prosemirror-tables';
+
+class PubTableView extends TableView {
+	constructor(node, whatever) {
+		super(node, whatever);
+		this.syncId(node);
+	}
+
+	update(node) {
+		super.update(node);
```

Comment:
⚠️ This should return the value of `super.update(node)` to indicate to Prosemirror that the node should (not?) be redrawn